vendor/assets/javascripts/remote_upload.js

```diff
@@ -0,0 +1,22 @@
+Cmsify.remoteUpload = function(form, elementToClone, elementToInsertBefore, callback) {
```

Comment:
Pass the elements in as selectors, then jQuery wrap them in the function. That way, the function will except either jQuery elements or selectors and behave in the same way.
```
Cmsify.remoteUpload($(this).find('form'), 
  '.js-featured-image-table-element',
  '.js-featured-image-table-element',
  function(clone, cloneInput, req, res) {
    $('.js-featured-image').attr('src', res.attachment.url);
  });
```

Then you should be able to just do:
var $clone = $(elementToClone).clone();
$clone.insertBefore($(elementToInsertBefore));